checks.d/couch.py

```diff
@@ -98,19 +101,23 @@ def get_data(self, server, instance):
 
         # Get the list of whitelisted databases.
         db_whitelist = instance.get('db_whitelist')
+        self.db_blacklist.extend(instance.get('db_blacklist',[]))
         whitelist = set(db_whitelist) if db_whitelist else None
-        databases = set(self._get_stats(url, instance))
+        databases = set(self._get_stats(url, instance)) - set(self.db_blacklist)
         databases = databases.intersection(whitelist) if whitelist else databases
 
         if len(databases) > self.MAX_DB:
-            self.warning('Too many databases, only the first %s will be checked.' % self.MAX_DB)
+            self.log.warning('Too many databases, only the first %s will be checked.' % self.MAX_DB)
```

Comment:
Nope we wan't to keep using self.warning here. It will display the warning in the info page